src/adapters/IDB.js

```diff
@@ -403,15 +489,15 @@ export default class IDB extends BaseAdapter {
    * @return {Promise}
    */
   async getLastModified() {
-    await this.open();
-    return new Promise((resolve, reject) => {
-      const { transaction, store } = this.prepare(undefined, "__meta__");
-      const request = store.get(`${this.storeName}-lastModified`);
-      transaction.onerror = event => reject(event.target.error);
-      transaction.oncomplete = event => {
-        resolve((request.result && request.result.value) || null);
-      };
-    });
+    try {
+      let entry;
+      await this.prepare("timestamps", store => {
+        store.get(this.cid).onsuccess = e => (entry = e.target.result);
+      });
+      return entry ? entry.value : null;
```

Comment:
This looks scary. if `entry` is a map (e.g. `{}`) the result of this line is `undefined`. Had it been Python, the output of `map.value if map else None` would be `None` if `map` was `{}`.